### PR TITLE
Translate doc

### LINKED ISSUE #867: Add a "Translate this Doc" link to docs footer
At the bottom of each doc page, we have these links:

screen shot 2017-11-17 at 3 30 28 pm

We should add another that says "Translate this Doc".

If the visitor's locale is the default English (US, Canadian, or otherwise), one of the following should happen:

a) link to Crowdin project landing page. This is very easy to implement but is not the best UX, as the person has to then pick a target language and re-navigate to the file in question:

b) display a modal (or a new page, or a `<select>` element) that prompts for desired target langauge, then redirects to the Crowdin page for that language/filename pair on Crowdin.

If the visitor's locale is **not the default**, clicking "Translate this Doc" should take them straight to their language/filename pair on Crowdin. 🎯 

I've created a module [get-crowdin-file-ids](https://github.com/zeke/get-crowdin-file-ids#readme) that we weave into the output of the `electron-i18n` module, so we generate these "straight to the right translation page" URLs from the website.

### PR DESCRIPTION
**Pull request by pierreneter**
Close #867 
Related to #960, #910

### CODE REVIEW

**Comment by zeke** [comment]
@pierre I would prefer that this is handled in the electron-i18n repo and exported as a new property on the module, which you should now be able to work on without any secrets like `CROWDIN_KEY`. Does that make sense?

**Comment by zeke** [comment]
In here somewhere: https://github.com/electron/electron-i18n/blob/205a0e1be0506aacf01650df3523199e92907066/script/build-module.js#L40

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "dotenv": "^4.0.0",
     "electron-api-historian": "1.1.0",
     "electron-apps": "1.2214.0",
-    "electron-i18n": "0.61.0",
+    "electron-i18n": "github:electron/electron-i18n#translate-doc-url",
```
Comment (pierreneter) [inline package.json]: Related to: https://github.com/electron/electron-i18n/pull/218

**Comment by zeke** [comment]
I shipped https://github.com/electron/electron-i18n/pull/218/ and bumped the i18n version here.

**Comment by zeke** [comment]
I pushed up some fixes. This should be good to go!

Let's wait for the Heroku deployment and give it a go before we ship it.

**Comment by zeke** [comment]
This is great! If anyone wants to try it out:

1. Go to https://electron-website-pr-975.herokuapp.com/languages
1. Pick a language
1. Visit a doc, e.g. https://electron-website-pr-975.herokuapp.com/docs/api/browser-window
1. Scroll to the bottom
1. Click "Translate this doc" (or whatever this string is in your language)

You'll be taken directly to the right page on Crowdin! 🎉

**Comment by zeke** [comment]
@pierreneter is this ready to go? If so, please remove the WIP label so all checks will pass.

**Comment by vanessayuenn** [comment]
Hrmm 🤔 the links on the Chinese docs (and maybe other languages) are not working -- they're pointing to the wrong locale.

For example, https://electron-website-pr-975.herokuapp.com/docs/api/power-monitor, the "Translate this doc" link is pointing to `https://crowdin.com/translate/electron/92/en-zh-CN`.

**Comment by pierreneter** [comment]
Oooops. Sorry. I forgot

**Comment by pierreneter** [comment]
Fix it soob

**Comment by zeke** [comment]
Replaced by https://github.com/electron/electronjs.org/pull/1158 which is assembling URLs using the new [Crowdin File ID property](https://github.com/electron/electron-i18n/pull/230). Thanks, @pierreneter!

**Pull request closed without merging**